prowler/providers/gcp/gcp_provider.py

```diff
@@ -95,6 +122,10 @@ def projects(self):
     def project_ids(self):
         return self._project_ids
 
+    @property
+    def excluded_project_ids(self):
```

Comment:
Are you going to use this value outside the provider? If not you don't need to expose it as a property.

Reply:
Yes, I am using it when printing the credentials.